docs/sdks/server-sdks/dotnet.md

```diff
@@ -49,60 +75,262 @@ internal class AssignmentLogger : IAssignmentLogger
 :::
 
-## 3. Assign variations
+### Assignment functions
 
-Assigning users to flags or experiments with a single `GetStringAssignment` function:
+Every Eppo flag has a return type that is set once on creation in the dashboard. Once a flag is created, assignments in code are made using the corresponding typed function: 
 
-```csharp
-var assignedVariation = eppoClient.GetStringAssignment("<SUBJECT-KEY>", "<FLAG-KEY>", {
-  // Optional Dictionary of subject metadata for targeting.
-});
+```cs
+GetBooleanAssignment(...)
+GetNumericAssignment(...)
+GetIntegerAssignment(...)
+GetStringAssignment(...)
+GetJSONAssignment(...)
 ```
 
-The `GetStringAssignment` function takes two required and one optional input to assign a variation:
+Each function has the same signature (except for the type of `defaultValue`) and returns the type in the function name. For booleans use `getBooleanAssignment`, which has the following signature:
+
+```cs
+public bool GetBooleanAssignment(
+    string flagKey, 
+    string subjectKey, 
+    Dictionary<string, object> subjectAttributes, 
+    bool defaultValue
+)
+```
+
+## Advanced Options
+### Polling Interval
+
+For additional control in server deployments, the `EppoClientConfig` class can be initialized with a custom interval to override the default of 30sec.
+
+```cs
+
+var config = new EppoClientConfig("YOUR-API-KEY", myAssignmentLogger)
+    {
+        PollingIntervalInMillis = 5000
+    };
+```
+
+## Assignment Log Schema
+
 
-- `subjectKey` - The entity ID that is being experimented on, typically represented by a uuid.
-- `flagOrExperimentKey` - This key is available on the detail page for both flags and experiments.
-- `targetingAttributes` - An optional map of metadata about the subject used for targeting. If you create rules based on attributes on a flag/experiment, those attributes should be passed in on every assignment call.
 
-Starting in version `v2.0.0` typed functions are available:
+The SDK will invoke the `LogAssignment` function with an `event` object that contains the following fields:
+
+| Field                                     | Description                                                                                                              | Example                             |
+| ----------------------------------------- | ------------------------------------------------------------------------------------------------------------------------ | ----------------------------------- |
+| `Experiment` (string)                     | An Eppo experiment key                                                                                                   | "recommendation-algo-allocation-17" |
+| `Subject` (string)                        | An identifier of the subject or user assigned to the experiment variation                                                | UUID                                |
```

Comment:
I wonder if we should have an example instead of just `UUID` (such as a nano ID `"V1StGXR8_Z5jdHi6B-myT"`)

Update: we do this later in the bandit assignment schema `"ed6f85019080"`